tests/natural_language/WriteGoodLintBearTest.py

```diff
@@ -141,8 +141,9 @@
                                                       invalid_files=(
                                                           illusion_file,),
                                                       settings={
-                                                        'allow_repeated_words':
-                                                            False})
+                                                          'allow_' +
+                                                          'repeated_words':
```

Comment:
this seriously doesn't fit in one line? I find this formatting worse than before